scripts/run_e2e_tests.py

```diff
@@ -150,7 +150,7 @@ def ensure_screenshots_dir_is_removed():
 
 def cleanup():
     """Kill the running subprocesses and server fired in this program."""
-    dev_appserver_path = '%s/dev_appserver.py' % common.GOOGLE_APP_ENGINE_HOME
+    dev_appserver_path = '%s/' % common.GOOGLE_APP_ENGINE_HOME
```

Comment:
There were processes by files like `_python_runtime.py`. So, I think we need to kill the processes by any file in this directory.

Reply:
You probably need a different name for this variable.

Reply:
Done